fulp_modules/things/toys/code/Vending_machine_code.dm

```diff
@@ -0,0 +1,7 @@
+/obj/machinery/vending/games/Initialize()
```

Comment:
why isn't this in fulp_modules >> container_init >> vending_init.dm?

Reply:
BECAUSE IT DOESNT EXIST YET SWAIN MERGE MY PR